knex/config/database.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const env = process.env.NODE_ENV || 'development'
```

Comment:
Do we need `NODE_ENV` things?

Reply:
again, this was an example online. i can pare down but not sure if it really matters